buildenv/jenkins/JenkinsfileBase

```diff
@@ -73,6 +73,7 @@ def setupEnv() {
 	env.BUILD_LIST = params.BUILD_LIST ? params.BUILD_LIST : ""
 	env.TAP_NAME = "${JOB_NAME}.tap"
 	env.USE_JRE=params.USE_JRE ? params.USE_JRE : false
+	env.FAIL_TEST = ""
```

Comment:
hi @YeeSkywalker - please change "FAIL_TEST" as the name, since it sounds like a directive.  What this environment variable is really representing/storing is the "FAILED_TESTLIST".